.husky/pre-commit

```diff
@@ -2,4 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
-tsc ./bin/codecov.ts --help > HELPFILE
```

Comment:
Did you mean to remove this?

Reply:
@drazisil-codecov yup, this was not producing what I wanted